src/wallet/wallet.h

```diff
@@ -595,6 +598,19 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     void GetKeyBirthTimes(std::map<CKeyID, int64_t> &mapKeyBirth) const;
 
+    /**
+      * ZKeys
+      */
+    //! Generates a new zaddr
+    CZCPaymentAddress GenerateNewZKey();
+    //! Adds spending key to the store, and saves it to disk
```

Comment:
Encrypted zkeys are not saved or loaded to the wallet at the moment...

This work was put to the side as there was discussion around changing the encryption algorithm (see #1207) which would have resulted in changes to the Crypter class.  I remember this was discussed at a meeting but I don't believe a final decision was reached over the encryption method.

To help users, I can add a note to the RPC documentation that in z9, passphrase protected wallets do not support zkeys.